tests/src/FunctionalJavascript/EmbedTestBase.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_oembed\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+use Drupal\Tests\oe_oembed\Traits\OembedTestTrait;
+
+/**
+ * Base class for all functional JS embed tests.
+ */
+abstract class EmbedTestBase extends WebDriverTestBase {
```

Comment:
Missed preparation for covering other entity types.

Reply:
It's not needed to prepare for all tests.